lib/appium_lib/android/helper.rb

```diff
@@ -202,13 +200,13 @@ def ids id
     # @param class_name [String] the class name to find
     # @param index [Integer] the index
     # @return [Element] the found element of type class_name
-    def ele_index class_name, index
+    def ele_index(class_name, index)
       results = tags(class_name)
       if index == 'last()'
         index = results.length
         index -= 1 if index >= 0
       else
-        raise 'Index must be >= 1' unless index >= 1
+        fail 'Index must be >= 1' unless index >= 1
```

Comment:
There is a longish discussion about fail and raise usage:

https://github.com/bbatsov/ruby-style-guide/issues/233 <https://github.com/bbatsov/ruby-style-guide/issues/233>

Of course, it's up to us which one to prefer. We can disable this cop if we agree that raise is better than fail.

> On 25 Apr 2015, at 19:35, bootstraponline <notifications@github.com> wrote:
> 
> In lib/appium_lib/android/helper.rb <https://github.com/appium/ruby_lib/pull/320#discussion_r29102930>:
> 
> >        results = tags(class_name)
> >        if index == 'last()'
> >          index = results.length
> >          index -= 1 if index >= 0
> >        else
> > -        raise 'Index must be >= 1' unless index >= 1
> > +        fail 'Index must be >= 1' unless index >= 1
> Is there a reason this fail instead of raise?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/appium/ruby_lib/pull/320/files#r29102930>.
>